proxy/coco/src/project/create/validation.rs

```diff
@@ -266,16 +267,36 @@ impl Repository {
         log::debug!("Creating rad remote");
         let mut git_remote = Self::existing_remote(repo, &url)?
             .map_or_else(|| Remote::rad_remote(url, None).create(repo), Ok)?;
-        Self::push_default(&mut git_remote, default_branch)?;
+        // Self::push_default(&mut git_remote, default_branch)?;
+        Self::push_branches(repo, &mut git_remote)?;
         Ok(())
     }
 
-    /// Push the default branch to the provided remote.
-    fn push_default(remote: &mut git2::Remote, default_branch: &OneLevel) -> Result<(), Error> {
-        let default: FlatRef<RefLike, _> =
-            FlatRef::head(PhantomData, None, default_branch.clone().into());
-        log::debug!("Pushing default branch '{}'", default);
-        remote.push(&[default.to_string()], None)?;
+    fn push_branches(repo: &git2::Repository, remote: &mut git2::Remote) -> Result<(), Error> {
+        let local_branches = repo
+            .branches(Some(git2::BranchType::Local))
+            .map_err(git::error::Error::from)?
```

Comment:
The explicit `.map_err()` is not needed. The `?` operator takes care of this.

```suggestion
            .branches(Some(git2::BranchType::Local))?
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1278/commits/f302bf0abec12937206a43538e17417c4572cce1